src/query-to-decoder.ts

```diff
@@ -191,7 +191,7 @@ export function decoderFor(def: OperationDefinition | FragmentDefinition, info:
     let info_type = info.getType()
     let isMaybe = false
     if (info_type instanceof GraphQLNonNull) {
-      info_type = info_type['ofType'];
+      //info_type = info_type['ofType'];
```

Comment:
This would break the case of Non-Nullable Lists. info_type is assigned what GraphQLNonNull is wrapped around so further checks may be made.